atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1363,12 +1363,32 @@ void WebContents::DidChangeVisibleSecurityState() {
   blink::WebSecurityStyle security_style =
     web_contents()->GetDelegate()->GetSecurityStyle(
```

Comment:
atom_api_web_contents is the delegate (and a subclass of CommonWebContentsDelegate) so you can just call `GetSecurityStyle` directly

Reply:
not actually part of this PR, but just noticed it anyway